src/helpers/helpers.dom.js

```diff
@@ -150,21 +150,41 @@ export function getMaximumSize(canvas, bbWidth, bbHeight, aspectRatio) {
   };
 }
 
+/**
+ * @param {import('../core/core.controller').default} chart
+ * @param {number} [forceRatio]
+ * @param {boolean} [forceStyle]
+ * @returns {boolean} True if redraw is needed.
+ */
 export function retinaScale(chart, forceRatio, forceStyle) {
-  const pixelRatio = chart.currentDevicePixelRatio = forceRatio || 1;
-  const {canvas, width, height} = chart;
+  const pixelRatio = forceRatio || 1;
+  const devHeight = Math.floor(chart.height * pixelRatio);
```

Comment:
I'm not sure what the `dev` prefix means? If it's `device`, I would use the whole word as prefix.

Reply:
Used `device` as prefix.